lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -263,39 +265,23 @@ def set_aspect(self, aspect, adjustable=None, anchor=None, share=False):
             'equalyz'   adapt the y and z axes to have equal aspect ratios.
             =========   ==================================================
 
-        adjustable : None or {'box', 'datalim'}, optional
-            If not *None*, this defines which parameter will be adjusted to
-            meet the required aspect. See `.set_adjustable` for further
-            details.
-
-        anchor : None or str or 2-tuple of float, optional
-            If not *None*, this defines where the Axes will be drawn if there
-            is extra space due to aspect constraints. The most common way to
-            specify the anchor are abbreviations of cardinal directions:
-
-            =====   =====================
-            value   description
-            =====   =====================
-            'C'     centered
-            'SW'    lower left corner
-            'S'     middle of bottom edge
-            'SE'    lower right corner
-            etc.
-            =====   =====================
```

Comment:
The `anchor` and `share` parameters should stay documented as long as they are not removed. You should change the content to
```
        anchor : None or str or 2-tuple of float, optional
            .. deprecated:: 3.11
                This parameter has no effect.

        share : bool, default: False
            .. deprecated:: 3.11
                This parameter has no effect.

```